### DISCUSSION TITLE
Cannot get rid of colima setting `DOCKER_HOST`

### DISCUSSION THREAD

**Discussion by nevotheless** [abiosoft/colima#593] [Q&A] [answered]
I have a bunch of issues right now with docker desktop and podman because i had colima installed and it still somehow sets the `DOCKER_HOST` env var. I did remove colima code from my ~/.zshrc and uninstalled colima completely. Unfortunately this isn't enough. 

I would like to know what i need to do in order to not have the `DOCKER_HOST` Env Var overwritten anymore.

**Comment by abiosoft** [comment, marked as answer]
Colima does not set `DOCKER_HOST` and only uses contexts.

You can view docker contexts with `docker context list` and you can remove Colima's context with `docker context rm colima`

By the way, the context is removed on `colima stop`, so something else must've been off for it to still persist in your case.

**Comment by nevotheless** [comment]
I found out that i had put the DOCKER_HOST Var to my fig config.
